# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../yaboot/no-etc-yaboot.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- yaboot-1.3.13/Makefile	2004-07-11 22:12:03.000000000 +0200
+++ yaboot-1.3.13-fixed/Makefile	2006-08-15 14:37:25.000000000 +0200
@@ -201,12 +201,6 @@
 	install -o root -g root -m 0644 man/ybin.8.gz ${ROOT}/${PREFIX}/${MANDIR}/man8/ybin.8.gz
 	install -o root -g root -m 0644 man/yaboot.conf.5.gz ${ROOT}/${PREFIX}/${MANDIR}/man5/yaboot.conf.5.gz
 	@gunzip man/*.gz
-	@if [ ! -e ${ROOT}/etc/yaboot.conf ] ; then						\
-		echo "install -o root -g root -m 0644 etc/yaboot.conf ${ROOT}/etc/yaboot.conf"; \
-		install -o root -g root -m 0644 etc/yaboot.conf ${ROOT}/etc/yaboot.conf;	\
-	 else											\
-		echo "/etc/yaboot.conf already exists, leaving it alone";			\
-	 fi
 	@echo
 	@echo "Installation successful."
 	@echo
